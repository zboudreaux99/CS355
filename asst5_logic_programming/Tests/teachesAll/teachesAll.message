Output line 0:
  <i>expected: </i><span class='expected'>teaches</span>(Beam, Rao,Abadam, Faiz) teaches(Beam, Rao,Abdel, Marvis) teache
  <i>observed: </i><span class='observed'>uncaught</span> exception: error(existence_error(procedure,teaches/2),top_level
on std err:
/home/zeil/bin/gprolog: 2: PATH: not found

