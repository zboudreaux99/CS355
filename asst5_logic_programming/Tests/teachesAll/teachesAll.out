uncaught exception: error(existence_error(procedure,teaches/2),top_level/0)
